README.md

```diff
@@ -98,29 +98,38 @@ curl -L https://raw.githubusercontent.com/knative/serving/v0.5.2/third_party/ist
 install knative
 
 ```bash
-kubectl apply \
-  --selector knative.dev/crd-install=true \
-  --filename https://github.com/knative/serving/releases/download/v0.5.2/serving.yaml \
-  --filename https://github.com/knative/build/releases/download/v0.5.0/build.yaml \
-  --filename https://github.com/knative/serving/releases/download/v0.5.2/monitoring.yaml \
-  --filename https://raw.githubusercontent.com/knative/serving/v0.5.2/third_party/config/build/clusterrole.yaml
+kubectl apply --selector knative.dev/crd-install=true \
```

Comment:
We have version 0.6.1 in `Gopkg.toml`. Should we use CRDs with the same version?

Reply:
I will replaced by  0.6.1